app/helpers/application_helper.rb

```diff
@@ -19,4 +19,16 @@ def become_a_manager(name, classes)
   def all_categories
    Category.order(:name)
   end
+
+  def get_site_root_site_link
+    "http://#{request.domain}:#{request.port}"
+  end
+
+  def show_form_errors(form_object)
+    content = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.